config-templates/google/gcp-compute-persistent-disk-csi-driver/1.8.0/deployment.yaml

```diff
@@ -0,0 +1,848 @@
+    spec:
+      selector:
+        matchLabels:
+          app: gcp-compute-persistent-disk-csi-driver
+          app.kubernetes.io/name: gcp-compute-persistent-disk-csi-driver
+          app.kubernetes.io/version: 1.8.0
+      template:
+        metadata:
+          labels:
+            app: gcp-compute-persistent-disk-csi-driver
+            app.kubernetes.io/name: gcp-compute-persistent-disk-csi-driver
+            app.kubernetes.io/version: 1.8.0
+          annotations:
+            autoUpdate: "AUTO-UPDATE-ENABLED"
+        spec:
+          containers:
+          - args:
+            - --v=5
+            - --csi-address=/csi/csi.sock
+            - --kubelet-registration-path=/var/lib/kubelet/plugins/pd.csi.storage.gke.io/csi.sock
+            env:
+            - name: KUBE_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+            image: k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.5.0
+            livenessProbe:
+              exec:
+                command:
+                - /csi-node-driver-registrar
+                - --kubelet-registration-path=/var/data/kubelet/plugins/pd.csi.storage.gke.io/csi.sock
+                - --mode=kubelet-registration-probe
+              initialDelaySeconds: 3
+            name: csi-driver-registrar
+            volumeMounts:
+            - mountPath: /csi
+              name: plugin-dir
+            - mountPath: /registration
+              name: registration-dir
+          - args:
+            - --v=5
+            - --endpoint=unix:/csi/csi.sock
+            - --run-controller-service=false
+            image: k8s.gcr.io/cloud-provider-gcp/gcp-compute-persistent-disk-csi-driver:v1.7.2
+            name: gce-pd-driver
+            securityContext:
+              privileged: true
+            volumeMounts:
+            - mountPath: /var/lib/kubelet
+              mountPropagation: Bidirectional
+              name: kubelet-dir
+            - mountPath: /csi
+              name: plugin-dir
+            - mountPath: /dev
+              name: device-dir
+            - mountPath: /etc/udev
+              name: udev-rules-etc
+            - mountPath: /lib/udev
+              name: udev-rules-lib
+            - mountPath: /run/udev
+              name: udev-socket
+            - mountPath: /sys
+              name: sys
+          hostNetwork: true
+          nodeSelector:
+            kubernetes.io/os: linux
+          priorityClassName: csi-gce-pd-node
+          serviceAccountName: csi-gce-pd-node-sa
+          tolerations:
+          - operator: Exists
+          volumes:
+          - hostPath:
+              path: /var/lib/kubelet/plugins_registry/
+              type: Directory
+            name: registration-dir
+          - hostPath:
+              path: /var/lib/kubelet
+              type: Directory
+            name: kubelet-dir
+          - hostPath:
+              path: /var/lib/kubelet/plugins/pd.csi.storage.gke.io/
```

Comment:
change this to /var/data

Reply:
Changed here: https://github.com/IBM/ibm-satellite-storage/pull/221/commits/0f41c691643168e11046fa58c0916fd334695a3d